web/tsconfig.json

```diff
@@ -2,17 +2,17 @@
     "extends": "./tsconfig.base.json",
     "compilerOptions": {
         "paths": {
-            "@goauthentik/authentik/*": ["src/*"],
```

Comment:
ESlint had never been run against this file. It complained that the "relative locations" had a poor syntax, so, now they don't.